src/CommandLineArgumentParser.c

```diff
@@ -15,25 +15,25 @@
 #include <string.h>
 #include "CommandLineArgumentParser.h"
 
-const char *CMD_ARG_HELP[3] = {
+const char *CMD_ARG_HELP[ARG_ARRAY_SIZE] = {
```

Comment:
naming use RTI_ as prefix (I said that in a previous comment). Also provide a more meaningful name: RTI_CMD_ARG_HELP_ELEMENT_COUNT (or something similar)

Reply:
Done.